examples/show_mipnerf360_allscales.py

```diff
@@ -0,0 +1,45 @@
+import json
```

Comment:
actually, another idea of managing these scripts is to have a `mip_splatting` folder under `examples`, in which you could have multiple scripts in there, without worrying about merging or conflicting with other scripts. I think this is a scalable way to manage the scripts when more things are supported in the future.

And you could also have a customized `simple_trainer.py` living in it, basically whatever you need to be customized. But this is less preferred because a customized `simple_trainer.py` means the mip-splatting feature would be disjoint with other future features for user to play with. So ideally we merge the trainer but could put the scripts into an isolated folder.